cmd/logs.go

```diff
@@ -3,11 +3,14 @@ package cmd
 import (
 		"fmt"
 		"strings"
+	  "time"
```

Comment:
FWIW, there are a couple other spots that need `go fmt` love as well. I just pointed it out here. Would probably be a good idea to do it either as part of the build or as a `pre-push` Git Hook.